samples/FallbackSendFile/SendFileMiddleware.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Features;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace FallbackSendFile
+{
+    /// <summary>
+    /// Middleware which adds an <see cref="IHttpSendFileFeature"/> implementation if currently missing.
+    /// </summary>
+    public class SendFileMiddleware
```

Comment:
This is coming from MVC/test/WebSites/FilesWebSite. The request in aspnet/Mvc#3612 was to move it elsewhere.

Could just delete this middleware and the feature implementation from MVC repo if you prefer. It's certainly unneeded in the MVC repo.